### DISCUSSION TITLE
question to bootup my first example with new PCB layout on Nano board

### DISCUSSION THREAD

**Discussion by rniueng** [felias-fogg/dw-link#7] [Q&A]
Hello Bernhard, 

It's very nice debugger tool. I would like to have it for my ATTiny13 and ATTiny85 projects debug. Firstly I created a new layout file for Ardunio Nano based on your schematic. After I verify this board without any issue, I'll share the design here.

![image](https://user-images.githubusercontent.com/61178189/192227916-508d8d06-f787-46ef-b347-503b6883b097.png)
[dw-probe-v1.1.pdf](https://github.com/felias-fogg/dw-link/files/9644415/dw-probe-v1.1.pdf)

I'm attempting to verify it with one ATTiny device to blink a LED in PlatformIO. I got some questions need your help.
1. In platformio.ini file, I modified the example at below line14 and line17&18. Please advise if these are connect.
![image](https://user-images.githubusercontent.com/61178189/192235818-16f1364c-01b4-459b-82d3-875fae6d634f.png)
2. Before the user start to verify debug, we may also want to verify ISP firstly. As my understanding at this time, this may controlled by dw-link automatically. But I'm not sure if the platformio.ini file should be modified at line11: default_envs = debug,  to change it to target env such as attiny85 in this example. Please advise.

I think to provide the entire PlatformIO project for the blink LED example will be very helpful for user.

Thanks,
rniu.

**Comment by felias-fogg** [comment]
Hi rniu,

sorry for the last answer! Looks cool. Did you send it to a PCB fab and build it?
